packages/mobile/src/components/core/FadeInView.tsx

```diff
@@ -0,0 +1,19 @@
+import type { ReactNode } from 'react'
+
+import Animated, { Easing, FadeIn } from 'react-native-reanimated'
+
+type FadeInViewProps = {
+  children: ReactNode
+}
+
+const fadeInConfig = FadeIn.easing(Easing.ease)
+  .withInitialValues({
+    opacity: 0.3
```

Comment:
Does 0.3 initial opacity look better with this new animation?

Reply:
yeah it does, starting at 0.5 it doesnt feel full enough. happy to tweak this stuff as we play with it?